.github/workflows/publish.yml

```diff
@@ -1,29 +1,86 @@
-name: brew pr-pull
+name: Publish Bottles
 on:
-  pull_request_target:
-    types:
-      - labeled
+  pull_request:
+    paths:
+      - Formula/*.rb
+
 jobs:
-  pr-pull:
-    if: contains(github.event.pull_request.labels.*.name, 'pr-pull')
-    runs-on: ubuntu-22.04
+  build-bottles:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: ${{ matrix.os }}
     steps:
       - name: Set up Homebrew
         uses: Homebrew/actions/setup-homebrew@master
```

Comment:
hmm...no tags or releases for this action...which makes it harder to pin and update on our schedule. Pinning to a commit doesn't seem like the best option here either...so I guess this will have to do.